languages/julia/exercises/concept/fibonacci-iterator/.meta/example.jl

```diff
@@ -0,0 +1,24 @@
+# Step 1
+struct Fib
+    n::Int
+end
+
+# Step 2
+function Base.iterate(fib::Fib, state=(i=1, vals=(1, 1)))
```

Comment:
The named tuple isn't strictly necessary but I personally find it neater to separate the counter and the last two values.